keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -157,13 +159,13 @@ COSI defines 5 new API types
  - [BucketClass](#bucketclass)
  - [BucketAccessClass](#bucketaccessclass)
 
-Detailed information about these API types are provided inline with user stories. 
+Detailed information about these API types are provided inline with user stories.
 
 Here is a TL;DR version:
 
- - BucketClaims/Bucket are similar to PVC/PV. 
- - BucketClaim is used to request generation of new buckets. 
- - Buckets represent the actual Bucket. 
+ - BucketClaims/Bucket are similar to PVC/PV.
+ - BucketClaim is used to request generation of new buckets.
+ - Buckets represent the actual Bucket.
```

Comment:
Looks like my editor is automatically cleaning up a bunch of whitespace all through the doc